src/components/html-text-input.js

```diff
@@ -100,17 +117,13 @@ export class HTMLInputView extends React.Component<PropsType, StateType> {
 						ref={ ( textInput ) => this.textInput = textInput }
 						textAlignVertical="top"
 						multiline
-						style={ { ...styles.htmlView, height: this.state.contentHeight + 16 } }
```

Comment:
`contentHeight` is not being used anymore. It would be good to remove it from the state.

Reply:
And being at it, it looks like the `textInput` ref is also not being used anymore. This might be a good opportunity to clean it up :)

Reply:
All true!

As far as I see textInput was even not used before this PR :)